php-frontend/src/main/java/org/sonar/php/tree/impl/declaration/ClassDeclarationTreeImpl.java

```diff
@@ -124,6 +125,27 @@ public SyntaxToken closeCurlyBraceToken() {
     return closeCurlyBraceToken;
   }
 
+  @Nullable
+  @Override
+  public MethodDeclarationTree fetchConstructor() {
+    MethodDeclarationTree PHP5 = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00117)